UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -248,6 +249,9 @@ public BleedingState CalculateBleedingState()
 
 		public PlayerScript playerScript;
 
+		public event Action<DamageType> OnTakeDamageType;
+		public UnityEvent OnLowHealth = new UnityEvent();
```

Comment:
you have an example of c# event just above the one you did, why not vanilla c# events instead.